docs: Grammar overhaul (#20636)
nuxt/nuxt: docs/2.guide/2.directory-structure/1.middleware.md

Nuxt provides two globally available helpers that can be returned directly from the middleware:

1. `navigateTo (to: RouteLocationRaw | undefined | null, options?: { replace: boolean, redirectCode: number, external: boolean )` - Redirects to the given route within plugins or middleware. It can also get called directly to perform page navigation.
2. `abortNavigation (err?: string | Error)` - Aborts the navigation with an optional error message.

> Unlike navigation guards in [the vue-router docs](https://router.vuejs.org/guide/advanced/navigation-guards.html#global-before-guards), a third `next()` argument is not passed, and redirects or route cancellation is handled by returning a value from the middleware. Possible return values are:

Reviewer: ```suggestion
Unlike navigation guards in the [Vue Router docs](https://router.vuejs.org/guide/advanced/navigation-guards.html#global-before-guards), a third `next()` argument is not passed, and redirect or route cancellation is handled by returning a value from the middleware. Possible return values are:
```

::

::alert{type=warning}

> We recommend using the helper functions above for performing redirects or stopping navigation. Other possible return values described in [the vue-router docs](https://router.vuejs.org/guide/advanced/navigation-guards.html#global-before-guards) may work, but there may be breaking changes in the future.

Reviewer: ```suggestion
We recommend using the helper functions above for performing redirects or stopping navigation. Other possible return values described in the [Vue Router docs](https://router.vuejs.org/guide/advanced/navigation-guards.html#global-before-guards) may work, but there may be breaking changes in the future.
```

::

## What Order Middleware Runs In

Middleware runs in the following order:

> 1. Global Middleware

Reviewer: ```suggestion
1. Global middleware
```

> 2. Page-defined middleware order (if there are multiple middlewares declared with the array syntax)

Reviewer: Thoughts welcome! That _order_ word before opening bracket didn't really fit nicely.
```suggestion
2. Page-defined middleware (Multiple middleware declared with the array syntax run in the order in which they are declared.)
```

</script>
```

> Now, before the navigation to that page can complete, the `auth` route middleware will be ran.

Author: You're right, changed to will run
